bpf/lib/lb.h

```diff
@@ -1023,7 +1028,9 @@ static __always_inline int lb4_extract_key(struct __ctx_buff *ctx __maybe_unused
 	/* FIXME: set after adding support for different L4 protocols in LB */
 	key->proto = 0;
 	key->address = (dir == CT_INGRESS) ? ip4->saddr : ip4->daddr;
-	csum_l4_offset_and_flags(ip4->protocol, csum_off);
+	if (!ipv4_is_not_first_fragment(ip4)) {
+		csum_l4_offset_and_flags(ip4->protocol, csum_off);
+	}
```

Comment:
Checkpatch will complain about stuff like this because kernel style is to omit the braces for single-statement conditionals. This make target is made for your convenience to check on stuff like this:

```
$ make -C bpf checkpatch
```

Reply:
the command is handy. Thanks